brd-ios/breadwallet/src/Views/SendViewCells/AttributeCell.swift

```diff
@@ -45,6 +45,15 @@ class AttributeCell: UIView {
     fileprivate let gr = UITapGestureRecognizer()
     fileprivate let tapView = UIView()
     private let border = UIView(color: .secondaryShadow)
+    
+    lazy var infoButton: UIButton = {
+        let infoButton = UIButton()
+        infoButton.setImage(UIImage(named: "Faq"), for: .normal)
```

Comment:
@rcresnik @dangelovska would probably be nice to have R.swift or something similar at least for the images.